Packs/PAN-OS/pack_metadata.json

```diff
@@ -2,7 +2,7 @@
     "name": "PAN-OS",
     "description": "Manage Palo Alto Networks Firewall and Panorama. For more information see Panorama documentation.",
     "support": "xsoar",
-    "currentVersion": "1.10.0",
+    "currentVersion": "1.10.1",
```

Comment:
update it in a minor update (should be ```1.11.0```)

Reply:
Fixed